impeller/renderer/backend/vulkan/swapchain/ahb/ahb_swapchain_impl_vk.cc

```diff
@@ -130,6 +131,11 @@ bool AHBSwapchainImplVK::Present(
     return false;
   }
 
+  auto context = transients_->GetContext().lock();
+  if (context) {
+    ContextVK::Cast(*context).GetGPUTracer()->MarkFrameEnd();
+  }
+
```

Comment:
Right, I tried not to include the final cmd buffer submission because I don't want to measure anything related to presentation time. In theory it doesn't matter because the probe itself is using bottom of pipe